src/otx/core/exporter/base.py

```diff
@@ -94,6 +143,59 @@ def to_onnx(
         Returns:
             Path: path to the exported model.
         """
+        import io
+        import json
+        import os
+        import tempfile
+        from zipfile import ZipFile
+
+        from otx.core.exporter.exportable_code import demo
+
+        work_dir = Path(demo.__file__).parent
+        parameters: dict[str, Any] = {}
+        if self.metadata is not None:
+            parameters["type_of_model"] = self.metadata.get(("model_info", "task_type"), "")
+            parameters["converter_type"] = self.metadata.get(("model_info", "model_type"), "")
+            parameters["model_parameters"] = {
+                "labels": self.metadata.get(("model_info", "labels"), ""),
+                "labels_ids": self.metadata.get(("model_info", "label_ids"), ""),
+            }
+        zip_buffer = io.BytesIO()
+        temp_dir = tempfile.TemporaryDirectory()
+        with ZipFile(zip_buffer, "w") as arch:
+            # model files
+            path_to_model = self.to_openvino(model, Path(temp_dir.name), base_model_name, precision)
+            arch.write(str(path_to_model), Path("model") / "model.xml")
+            arch.write(str(path_to_model)[:-4] + ".bin", Path("model") / "model.bin")
+
+            arch.writestr(
+                str(Path("model") / "config.json"),
+                json.dumps(parameters, ensure_ascii=False, indent=4),
+            )
+            # python files
+            arch.write(
+                Path(work_dir) / "requirements.txt",
```

Comment:
```suggestion
                work_dir / "requirements.txt",
```
`work_dir` is already Path.